CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 2024-03-14
+
+- Shell hardening ([#115](https://github.com/heroku/heroku-buildpack-apt/pull/115))
+- Handle multi-packages lines when capturing buildpack metadata ([#112](https://github.com/heroku/heroku-buildpack-apt/pull/112))
```

Comment:
```suggestion
- Handle multi-package lines when capturing buildpack metadata ([#112](https://github.com/heroku/heroku-buildpack-apt/pull/112))
```
Might sound a bit better (or alternatively "Handle lines with multiple packages when capturing buildpack metadata")?

Reply:
Oops, Thanks for catching that typo 😄